tools/fast/fasting/basic.go

```diff
@@ -0,0 +1,132 @@
+package fasting
+
+import (
+	"context"
+	"io/ioutil"
+	"math/big"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-ipfs-files"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/testhelpers"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+	"github.com/filecoin-project/go-filecoin/tools/fast/series"
+	localplugin "github.com/filecoin-project/go-filecoin/tools/iptb-plugins/filecoin/local"
+)
+
+type Environment struct {
+	fast.Environment
+
+	t   *testing.T
+	ctx context.Context
+
+	pluginName string
+	pluginOpts map[string]string
+
+	fastenvOpts fast.EnvironmentOpts
+
+	GenesisMiner *fast.Filecoin
+}
+
+func (env *Environment) NewNode() *fast.Filecoin {
```

Comment:
What about `RequireNewNode`? It will help follow the pattern already established in the [message queue tests](https://github.com/filecoin-project/go-filecoin/blob/master/core/message_queue_test.go), and reads better I think.

Reply:
SGTM same answer here as above